### PR TITLE
Depend on sqlite3 `~> 1.4`.

### PR DESCRIPTION
**Pull request by ioquatix**
Try to fix `turbo-rails` tests.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,6 +28,8 @@ gem "rackup" if use_rackup
 
 gem "jruby-openssl", :platform => "jruby"
 
+gem "sqlite3", "~> 1.4"
```
Comment (dentarg) [inline Gemfile:31]: Add a comment on why we have this? Or make it conditional/opt-in? (ENV that the workflow can set) 

Very few run the turbo rails test when they clone puma repo and run tests, and as far as I know, you don't need SQLite then.

**Comment by MSP-Greg** [comment]
I've got this passing, give me a minute...

File: Gemfile (hunk 1 above)
Comment (ioquatix) [reply Gemfile:31]: CI currently appears to be broken, is it not important to fix that?

**Comment by ioquatix** [comment]
Sure, feel free to add commits to this branch.

**Comment by ioquatix** [comment]
Closing in favour of #3401

**Comment by MSP-Greg** [comment]
> Sure, feel free to add commits to this branch.

Sorry, busy morning, working on some other things.  This did seem to demand a quick fix, so...

**Pull request closed without merging**